scripts/bundle.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-const child_process = require('child_process');
+const { spawn } = require('child_process');
 const fs = require('fs');
```

Comment:
The destructuring doesn't really clean the code, rather obfuscates it.
There are different standards for a library code which is supposed to be read and debugged by many, and a proprietary in-house code which can be (temporarily) hacked using syntax shortcuts.